packages/theme/pages/Checkout/Payment.vue

```diff
@@ -148,18 +149,21 @@ export default {
     const processOrder = async () => {
       const orderId = orderGetters.getId(cart.value);
       try {
-        await savePayment.value();
+        paymentSuccessful.value = await savePayment.value();
       } catch (e) {
         Logger.error(e);
         return;
       }
 
-      await make();
-      if (makeError.value.make) {
-        Logger.error(makeError.value.make);
-        return;
+      // Only complete order if payment is successful
```

Comment:
We should avoid such obvious comments. The code itself comments on this logic quite well. I'd suggest removing this comment.

Reply:
No problem. Removed.